iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/AttachLoadOnVoltageLevel.java

```diff
@@ -0,0 +1,171 @@
+        if (voltageLevel.getNodeBreakerView().getBusbarSectionCount() > 1) {
+            BusbarSectionPosition position = bbs.getExtension(BusbarSectionPosition.class);
+            voltageLevel.getNodeBreakerView().getBusbarSectionStream().forEach(busbarSection -> {
+                if (busbarSection.getExtension(BusbarSectionPosition.class).getSectionIndex() == position.getSectionIndex() && !busbarSection.getId().equals(bbsId)) {
+                    createNodeBreakerDisconnector(forkNode, busbarSection.getTerminal().getNodeBreakerView().getNode(),
+                            String.valueOf(busbarSection.getId()), loadId, voltageLevel.getNodeBreakerView(), true);
+                }
+            });
+        }
+    }
+
+    private void createTopologyFromBusbarList(Network network, VoltageLevel voltageLevel, int loadNode, int forkNode) {
+        createNodeBreakerBreaker(loadNode, forkNode, "", loadId, voltageLevel.getNodeBreakerView(), false);
+        bbsIds.stream().forEach(id -> {
+            BusbarSection bbs = network.getBusbarSection(id);
+            int bbsNode = bbs.getTerminal().getNodeBreakerView().getNode();
+            createNodeBreakerDisconnector(forkNode, bbsNode, String.valueOf(bbsNode), loadId, voltageLevel.getNodeBreakerView(), true);
+        });
+    }
+
+    private void createTopologyFromVoltageLevel(VoltageLevel voltageLevel, int loadNode, int forkNode) {
+        BusbarSection bbs = voltageLevel.getNodeBreakerView().getBusbarSectionStream().reduce((first, second) -> first).orElse(null);
+        if (bbs != null) {
+            createNodeBreakerSwitches(loadNode, forkNode, bbs.getTerminal().getNodeBreakerView().getNode(), loadId, voltageLevel.getNodeBreakerView());
+        }
+    }
+
+    @Override
+    public void apply(Network network, ComputationManager computationManager) {
+        apply(network);
+    }
+
+    @Override
+    public void apply(Network network) {
+        VoltageLevel voltageLevel = network.getVoltageLevel(voltageLevelId);
+        if (voltageLevel == null) {
+            throw new PowsyblException(String.format("Voltage level %s is not found", voltageLevelId));
+        }
+
+        TopologyKind topologyKind = voltageLevel.getTopologyKind();
+        if (topologyKind != TopologyKind.NODE_BREAKER) {
+            throw new PowsyblException(String.format("Voltage level %s is not in node/breaker.", voltageLevelId));
+        }
+
+        //Create the new load
+        LoadAdder loadAdder = createLoadAdder(loadId, loadType, p0, q0, network.getVoltageLevel(voltageLevelId));
+
+        int loadNode = voltageLevel.getNodeBreakerView().getMaximumNodeIndex() + 1;
+        int forkNode = loadNode + 1;
+        loadAdder.setNode(loadNode).add();
+
+        //Create a switch and a breaker linking the load to the busbar sections
+        if (bbsId != null) {
+            createTopologyAutomatically(network, voltageLevel, loadNode, forkNode);
+        }
+
+        if (bbsIds != null) {
```

Comment:
If someone enters a list that contains only one element, what behavior should this have?